Gems/LmbrCentral/Code/Source/Builders/LuaBuilder/LuaBuilderWorker.h

```diff
@@ -7,15 +7,18 @@
  */
 #pragma once
 
-#include <AzCore/Component/Component.h>
-#include <AzCore/Outcome/Outcome.h>
 #include <AssetBuilderSDK/AssetBuilderBusses.h>
 #include <AssetBuilderSDK/AssetBuilderSDK.h>
+#include <AzCore/Component/Component.h>
+#include <AzCore/Outcome/Outcome.h>
+namespace AZ
+{
+    class ScriptContext;
+}
 
 namespace LuaBuilder
 {
-    class LuaBuilderWorker
-        : public AssetBuilderSDK::AssetBuilderCommandBus::Handler
+    class LuaBuilderWorker : public AssetBuilderSDK::AssetBuilderCommandBus::Handler
```

Comment:
The format of declaring the base class was correct in the previous diff

Reply:
will update, clang-format caused this, using our `.clang-format` file

I wonder if we can use `BreakBeforeInheritanceComma: true` to preserve such formatting